modules/metrics/src/main/java/org/apache/ignite/internal/metrics/DistributionMetric.java

```diff
@@ -59,18 +60,34 @@ public class DistributionMetric extends AbstractMetric implements CompositeMetri
     /** Bounds of measurements. */
     private final long[] bounds;
 
+    /** Value string formatter or {@code null} to use default formatter. */
+    private final Function<long[], String> stringFormatter;
+
     /** List of scalar metrics. */
     private volatile List<Metric> scalarMetrics = null;
 
     /**
-     * The constructor.
+     * Constructor.
```

Comment:
You removed the article but added it a few lines below :)

Reply:
This one is just a call to another constructor.
I am not sure about the usage of article hire, but I think the only constructor can be marked with the artical 'the'.